include/aspect/geometry_model/interface.h

```diff
@@ -315,10 +315,15 @@ namespace aspect
          * have reasonably crossed a periodic boundary. If so, it will adjust the position
          * as described by the periodic boundary (e.g. a translation in a box, or a rotation
          * in a spherical shell). Afterwards, if it adjusted @p position, it will also adjust
-         * all locations given in @p connected_positions (if any where given) in the same way.
-         * Adjusting @p connected_positions allows to adjust related temporary variables,
+         * all locations given in @p connected_positions (if any where given) and all
+         * velocities given in @p connected_velocities in the same way.
+         * Adjusting both of these allows to adjust related temporary variables,
          * e.g. the intermediate results of an ordinary differential equation solver
-         * that are used to compute differences/directions between points.
+         * that are used to compute differences/directions between points. The reason
+         * positions and velocities have to be treated separately is that some geometries
+         * have to adjust them differently across a periodic boundary, e.g. a box has
+         * to translate positions but not velocities, while a spherical shell has to
+         * rotate both.
```

Comment:
The movement of points can be facilitated by a rotation, but I think it's a bit confusing to say that they both have to be "rotated". Perhaps say that the points have to be moved and the velocities rotated?